source/iterators.tex

```diff
@@ -4406,7 +4406,7 @@
 \begin{itemdescr}
 \pnum
 \constraints
-The expression \tcode{x + n} shall be valid and have type \tcode{Iterator}.
+The expression \tcode{x + n} shall be valid.
```

Comment:
The intent here is "The expression `\tcode{x.base() + n}` shall be valid and have type `\tcode{Iterator}`." Admittedly it's weird to constrain `n + move_iterator` when `move_iterator + n` isn't constrained; I'll open an LWG issue to constrain the member operators similarly.

Reply:
...and while I'm here, thanks for the incredibly detailed review of and corrections to the Ranges wording.

Reply:
> The intent here is "The expression `\tcode{x.base() + n}` shall be valid and have type `\tcode{Iterator}`." Admittedly it's weird to constrain `n + move_iterator` when `move_iterator + n` isn't constrained; I'll open an LWG issue to constrain the member operators similarly.

Right. I hoped that [move.iter.requirements] did that job.

> ...and while I'm here, thanks for the incredibly detailed review of and corrections to the Ranges wording.

Sure :)